packages/server/src/course/course.controller.ts

```diff
@@ -73,4 +74,22 @@ export class CourseController {
 
     return queue;
   }
+
+  @Delete(':id/ta_location/:room')
```

Comment:
I mentioned this in the other PR but it might better to have the room in the payload rather than the url. It works either way though, I'm curious what other people think